ai-jam-js/maybe_download_mags.py

```diff
@@ -12,8 +12,8 @@
 for mag_file in mag_files:
   output_file = mag_file.split('/')[-1]
   if os.path.exists(output_file):
-    print 'File %s already present' % mag_file
+    print('File %s already present' % mag_file)
```

Comment:
Can you add the standard future imports at the top of the file and test that this still works under python 2?

```
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
```